### DISCUSSION TITLE
If I have multiple GPUs, how can I choose any of them #5149

### DISCUSSION THREAD

**Discussion by wanghr323** [microsoft/onnxruntime#5880] [Other Q&A]
**Describe the bug**
not a bug

python api.
I want to know if i have multiple GPUS how can i specify which gpu to use in inference session?

**Comment by pranavsharma** [comment]
https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/python/onnxruntime_test_python.py#L66
